### DISCUSSION TITLE
running docker compose import or cp cannot find the folder

### DISCUSSION THREAD

**Discussion by krisatmanning** [photoprism/photoprism#4091] [Q&A] [answered]
When I run docker compose exec photoprism import <folder> and the folder contains a capitalized letter (such as Photos/) you receive an error because it can't find "photos/". 

EDIT: it appears not related to capitalization because when I changed from ./Photos to ./photos the same error was returned.

`
kris@blade:~$ sudo docker compose exec photoprism photoprism import /srv/smb/nextcloud/kris/files/Photos
INFO[2024-03-06T18:21:57Z] Become a member today, support our mission and enjoy our member benefits! 💎 
INFO[2024-03-06T18:21:57Z] Visit https://www.photoprism.app/membership to learn more. 
INFO[2024-03-06T18:21:57Z] moving media files from /srv/smb/nextcloud/kris/files/Photos to /photoprism/originals 
ERRO[2024-03-06T18:21:57Z] import: directory /srv/smb/nextcloud/kris/files/photos not found 
INFO[2024-03-06T18:21:57Z] completed in 17.196548ms      
`

The same issue occurs with other folder-based statements such as cp.

No images show up in the originals folder either (mine is set to /srv/photoprism/originals)

The source folder was tested with chmod 777 and +x with no change in behavior.

To be sure, here's some output. If this is a permissions issue, I can't for the life of me think what could be wrong:
<img width="796" alt="image" src="https://github.com/photoprism/photoprism/assets/112195858/7d9c7c81-50a1-4b4c-9565-9575d4b8630d">

This is build Build [231128-f48ff16ef] on Ubuntu 22 LTS

**Comment by lastzero** [comment]
Does it work with lowercase path names? It could otherwise just be a logging issue where the names are displayed in lowercase while the actual operations are case sensitive. Note that paths mounted on the host won't automatically be visible in Docker containers.

**Reply by krisatmanning** [reply]
Hi lastzero...yes I have tried making the path lower case and get the same error...so probably a log message thing. 

I've got this currently in my yml file:

<img width="796" alt="image" src="https://github.com/photoprism/photoprism/assets/112195858/3d439ba3-f529-45bc-9bdd-36b33c2f03da">

I've tried running as root and as 33 (www-data). root works fine for starting up (still can't import), but www-data makes it so the docker never fully starts. 

Import also doesn't work from Library -> Index

I have many pictures in ./photos that I'd like to start with...and I'm assuming that's what import is for.... I just can't get it to work for some reason.

**Reply by krisatmanning** [reply]
I have also reset Mariadb, deleted the containers and images...volumes. But it does not help.

**Reply by krisatmanning** [reply]
Does it matter if it's on a samba share?

**Reply by krisatmanning** [reply]
More information:  When I get a bash prompt from the docker container, I see this:
<img width="457" alt="image" src="https://github.com/photoprism/photoprism/assets/112195858/51e3282e-cc68-435c-b2cd-8c77ef31f075">

so it makes the target location's subpath (./photos) lowercase.

Consider I see plenty of files in that location (as you can see in the image) then why do they not appear in the web UI? Shouldn't they have automatically been loaded into originals?

in the container /photoprism/originals is blank

**Reply by krisatmanning** [reply]
Oh ok, I think I'm getting somewhere. The import command needs to point to a path inside the container?  Then how did they get to the container's import directory in the first place?

**Reply by lastzero** [reply, marked as answer]
If you mounted it as import path, you can provide the subfolder path as argument, so NOT the absolute path on your host.

**Reply by krisatmanning** [reply]
Ohhh ok. let me try then. so since my pictures are all under ./photos and I've set the import as /srv/..../photos then .... then I guess I just use ./ as the import path?

**Reply by krisatmanning** [reply]
Thank you. I also realized that you don't even need to provide a path and it uses the mapping for the docker-compose.yml file. All is fine now. Thanks lastzero.
